libs/blocks/merch/autoblock.js

```diff
@@ -35,9 +35,62 @@ export async function decorateCardCtasWithA11y(element, self) {
   });
 }
 
+export function handleCustomAnalyticsEvent(eventName, element) {
+  let daaLhValue = '';
+  let daaLhElement = element.closest('[daa-lh]');
+  while (daaLhElement) {
+    if (daaLhValue) {
+      daaLhValue = `|${daaLhValue}`;
+    }
+    const daaLhAttrValue = daaLhElement.getAttribute('daa-lh');
+    daaLhValue = `${daaLhAttrValue}${daaLhValue}`;
+    daaLhElement = daaLhElement.parentElement.closest('[daa-lh]');
+  }
+  if (daaLhValue) {
+    console.log('daaLhValue: ', `${eventName}|${daaLhValue}`);
+    // eslint-disable-next-line no-underscore-dangle
+    window._satellite?.track('event', {
+      xdm: {},
+      data: { web: { webInteraction: { name: `${eventName}|${daaLhValue}` } } },
+    });
+  }
+}
+
+export function getTabNestedLh(el) {
+  const tabPanel = el.closest('.tabpanel');
+  const tabDaaLh = tabPanel?.getAttribute('data-nested-lh');
+  return tabDaaLh ? `${tabDaaLh}--tab` : '';
+}
+
+function enableCardAnalytics(el, self) {
+  const tabPanel = el.closest('.tabpanel');
+  const tabDaaLh = tabPanel?.getAttribute('data-nested-lh') ? `${tabPanel.getAttribute('data-nested-lh')}--tab` : '';
+
+  const cards = self ? [el] : el.querySelectorAll('merch-card');
+  cards.forEach(async (card) => {
+    await card.checkReady();
+    card.setAttribute('name', card.getAttribute('daa-lh'));
```

Comment:
are we sure about this? isn't name used for something else? is it useful to have daa-lh dubbed this way?

Reply:
there is no 'name' on M@Sv2 card at the moment. might as well name them by product code.
i didn't find any usage except outdated one on catalog, but that one will still work as before.
alternatively i could use attribute 'analytics-id', is it better?

Reply:
daa-lh is not dubbed, from merch-card-collection cards it is actually removed